GestureHandler.js

```diff
@@ -400,15 +406,165 @@ const LongPressGestureHandler = createHandler(
   },
   {}
 );
+
+function backwardCompatibleTransformPanProps(props) {
```

Comment:
Don't like this naming. I suggest that we split this method into two: `transformPanGestureHandlerProps` and `validatePanGestureHandlerProps`

The first one will handle transforming while the second one will be called from inside __DEV__ block and perform all the checks.